set.mm

```diff
@@ -38280,6 +38280,19 @@ technically classes despite morally (and provably) being sets, like ` 1 `
   eqimss2 $p |- ( B = A -> A C_ B ) $=
     ( wss eqimss eqcoms ) ABCABABDE $.
 
+  ${
+    eqimssd.1 $e |- ( ph -> A = B ) $.
+    $( Equality implies inclusion, deduction version.  (Contributed by SN,
+       6-Nov-2024.) $)
+    eqimssd $p |- ( ph -> A C_ B ) $=
+      ( ssid eqsstrdi ) ABCCDCEF $.
+
+    $( Equality implies inclusion, deduction version.  (Contributed by SN,
+       15-Feb-2025.) $)
+    eqimss2d $p |- ( ph -> B C_ A ) $=
```

Comment:
I think we should use the label `eqimsscd` ("c" for "commuted", instead of "2"), see discussions in PR and PR #4676.
Here, the LHS and RHS of the subset relation are commuted.

Reply:
Agree. I'll also try to use this convention in the future.

Reply:
Done in https://github.com/metamath/set.mm/pull/4675/commits/4bb97b4243ab5ecf93b649e35c022690fb494800